### PR TITLE
Stdlib package

### PR DESCRIPTION
**Pull request by robin-da**
We ship the daml-stdlib as a package instead of a source code include.

### CODE REVIEW

**Comment by moritzkiefer-da** [comment]
Did you intend to duplicate the whole stdlib and daml-prim sourcecode?

**Comment by robin-da** [comment]
@moritzkiefer-da Yes for testing it now. I'll change it to a nice bazel rule in a minute.

File: daml-foundations/daml-ghc/daml-prim-and-stdlib-src/BUILD.bazel [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+
+filegroup(
+  name = "daml-prim-and-stdlib-src",
+  srcs = glob(["*.daml", "**/*.daml"]),
```
Comment (martin-drhu-da) [inline daml-foundations/daml-ghc/daml-prim-and-stdlib-src/BUILD.bazel]: Doesn't `**/*.daml` already cover `*.daml`?

File: daml-foundations/daml-ghc/package-database/BUILD.bazel [hunk 2]
```diff
@@ -123,9 +124,25 @@ daml_package_db(
   visibility = ["//visibility:public"]
 )
 
+daml_package_db(
+  name = "package_db_for_damlc",
+  pkgs =  [":daml-prim-1.0"
```
Comment (martin-drhu-da) [inline daml-foundations/daml-ghc/package-database/BUILD.bazel]: Could we have a list of DAML-LF versions somewhere in this file and generate this of packages list from the list of versions? We could also reuse the list of versions a bit further below.

File: daml-foundations/daml-ghc/package-database/deprecated/BUILD.bazel [hunk 3]
```diff
@@ -0,0 +1,67 @@
+# Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+load ("//daml-foundations/daml-ghc/package-database:util.bzl", "bundled_package_db", "daml_package_rule", "daml_package_db")
+
+bundled_package_db(
+    name = "package-db",
+    pkg_db = ":package_db_for_damlc",
+    daml_lf_versions = ["1.0", "1.1", "1.2", "1.3"],
```
Comment (martin-drhu-da) [inline daml-foundations/daml-ghc/package-database/deprecated/BUILD.bazel]: We could reuse the list of DAML-LF versions here again.

File: daml-foundations/daml-ghc/package-database/deprecated/BUILD.bazel [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+    visibility = ["//visibility:public"],
+)
+
+daml_package_rule(
```
Comment (martin-drhu-da) [inline daml-foundations/daml-ghc/package-database/deprecated/BUILD.bazel]: Can we make this a rule which is parameterised with the DAML-LF version?

Comment (robin-da) [reply daml-foundations/daml-ghc/package-database/deprecated/BUILD.bazel]: I think this rule creates just all targets. @moritzkiefer-da is cleaning up the package-db rules, can you include @martin-drhu-da suggestions in your PR?

**Pull request merged**